guides/Getting-Started.md

```diff
@@ -4,17 +4,31 @@ Let's walk through the basics of getting set up to use the PureScript compiler `
 
 We'll start with the installation of the compiler and Spago build tool, and then go through the basic usage of `purs repl`, working towards a solution of problem 1 from [Project Euler](http://projecteuler.net/problem=1).
 
-#### Installing the Compiler
+### Installing the Compiler
+
+#### Using NPM
 
 You'll need [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node) and to be [able to install global packages](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory) to proceed.
 
 The Purescript compiler (`purs`) can be installed with npm:
 
     npm install -g purescript
 
+#### Downloading Binaries
+
 (It can also be installed from [Hackage](http://hackage.haskell.org/package/purescript), or by downloading the latest [binary bundle](https://github.com/purescript/purescript/releases) for your OS. If you do so, make sure the `purs` executable is on your `$PATH`.)
 
-#### Setting up the Development Environment
+#### Using Homebrew
+
+    brew install purescript
+
+#### Using Nix
+
+You'll need [Nix](https://nixos.org/nix/) installed with a recent version of `nixpkgs` (19.09):
+
+    nix-env -i purescript
```

Comment:
May I suggest also adding [`easy-purescript-nix`](https://github.com/justinwoo/easy-purescript-nix)? Just starting learning PureScript, and it has everything a beginner needs (or any PS dev for that matter) - plus nothing is broken as it pins nixpkgs. [This stackoverflow thread](https://stackoverflow.com/questions/46094241/need-help-on-start-using-purescript-on-nixos) is a perfect summary how confusing the PS eco-system for a new-comer, adding NixOS to mix as well.

Reply:
It may be worth mentioning it not just in [INSTALL.md](https://github.com/purescript/purescript/blob/master/INSTALL.md) but somewhere here in this documentation repo, because `easy-purescript-repo` sets up a(n almost) complete development environment with the ["recommended" utilities](https://discourse.purescript.org/t/recommended-tooling-for-purescript-in-2020/1615).